src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const mainList = document.querySelector('ul');
+const listOfUlArray = Array.from(mainList.children);
+
+function toNumber(el) {
+  return Number(el.dataset.salary.slice(1).split(',').join(''));
+}
+
+function sortList(list) {
+  const sortLists = list.sort((a, b) => toNumber(b) - toNumber(a));
+
```

Comment:
The variable 'sortLists' is redundant since you can return the result of the sort function directly.